src/Commands/MakeFactoryReloadedCommand.php

```diff
@@ -7,14 +7,20 @@
 
 class MakeFactoryReloadedCommand extends GeneratorCommand
 {
+
```

Comment:
I noticed a lot of style changes as well. This makes it difficult to see the "real" code changes. I want to add a github action to take of style changes. This will definitely help.

Reply:
Yeah. Sorry. I use StyleCI myself so I don't pay a ton of attention to style as I go. Lazy old man habits. :)

Reply:
:-)